lib/api_catalogue.rb

```diff
@@ -10,6 +14,34 @@ def self.from_csv(catalogue_csv:, organisation_csv:)
     new(apis: apis, organisations: organisations)
   end
 
+  def self.from_url(url)
+    v1_alpha_fetcher = V1AlphaFetcher.new
+    FetcherService.new([v1_alpha_fetcher]).fetch(url)
```

Comment:
We could even have this generated by a static method on `FetcherService` like `FetcherService.instance` which pre-configures, but this works too